docs/vale-styles/Homebrew/Terms.yml

```
@@ -3,7 +3,8 @@ extends: substitution
 message: Use '%s' instead of '%s'.
 link: "https://github.com/Homebrew/brew/blob/HEAD/docs/Prose-Style-Guidelines.md#terminology-words-and-word-styling"
 level: error
-scope: $paragraph
+scope: paragraph
```

```suggestion
scope: paragraph & ~link
```
This will skip linting link text, e.g. "How to Open a Homebrew Pull Request".

---

@EricFromCanada Good catch! I think it'd be good to still do link text, though, so have handled the only case we don't care about there with a negative lookahead.